suite-native/feature-flags/src/useIsDeviceConnectEnabled.tsx

```diff
@@ -0,0 +1,18 @@
+import { useAtom } from 'jotai';
+
+import { atomWithUnecryptedStorage } from '@suite-native/storage';
+import { isDevelopOrDebugEnv } from '@suite-native/config';
+
+const isDeviceConnectEnabledAtom = atomWithUnecryptedStorage<boolean>(
+    'isDeviceConnectEnabledAtom',
+    isDevelopOrDebugEnv(),
+);
+
+export const useIsDeviceConnectEnabled = () => {
```

Comment:
What about suffixing/prefixing name always with `feature` => `useIsDeviceConnectFeatureEnabled` to be explicit that it's feature flag.

Reply:
https://github.com/trezor/trezor-suite/pull/9450/commits/a6b50cc45117d8b0cb492ed7ed025863bbf72e78